### PR TITLE
test(visual): add theme to notification banner test name

### PR DESCRIPTION
**Pull request by ozyx**
No related issue.

### Describe your changes:

- Fixes[ issue with Notification Banner visual test](https://app.circleci.com/pipelines/github/nasa/openmct/16951/workflows/2eed132a-6189-4150-a58a-30e207b1fc16/jobs/68717?invite=true#step-106-1278) not having a unique snapshot name
- Updates test selectors in FaultManagement visual tests to be more deterministic-- attempting to solve for [this visual test flake
](https://app.circleci.com/pipelines/github/nasa/openmct/16961/workflows/96a3ff98-961f-4e31-9643-88a44c3aa20d/jobs/68773?invite=true#step-106-1288)
- Moves "Planning" related visual tests to their own file: `planning.visual.spec.js`
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: e2e/helper/faultUtils.js [hunk 1]
```diff
@@ -141,8 +148,7 @@ async function clearSearch(page) {
  * @param {import('@playwright/test').Page} page
  */
 async function selectFaultItem(page, rowNumber) {
-    // eslint-disable-next-line playwright/no-force-option
-    await page.check(`.c-fault-mgmt-item > input >> nth=${rowNumber - 1}`, { force: true }); // this will not work without force true, saw this may be a pw bug
+    await page.locator(`.c-fault-mgmt-item > input >> nth=${rowNumber - 1}`).check();
```
Comment (unlikelyzero) [inline e2e/helper/faultUtils.js:151]: nice

File: e2e/tests/visual/planning.visual.spec.js [hunk 2]
```diff
@@ -24,29 +24,35 @@ const { test } = require('../../pluginFixtures');
...
         const plan = await createPlanFromJSON(page, {
-            json: examplePlanLarge
+            json: examplePlanSmall
         });
 
-        await setBoundsToSpanAllActivities(page, examplePlanLarge, plan.url);
-        await percySnapshot(page, `Plan View (theme: ${theme})`);
+        await setBoundsToSpanAllActivities(page, examplePlanSmall, plan.url);
+        await percySnapshot(page, `Plan View (theme: ${theme})`, {
+            scope: snapshotScope
```
Comment (unlikelyzero) [inline e2e/tests/visual/planning.visual.spec.js:42]: interesting

File: e2e/helper/faultUtils.js [hunk 3]
```diff
@@ -58,8 +58,15 @@ async function navigateToFaultManagementWithoutExample(page) {
 async function navigateToFaultItemInTree(page) {
     await page.goto('./', { waitUntil: 'networkidle' });
 
-    // Click text=Fault Management
-    await page.click('text=Fault Management'); // this verifies the plugin has been added
+    const mainTree = page.getByRole('tree', {
```
Comment (unlikelyzero) [inline e2e/helper/faultUtils.js:61]: can we just chain the selectors at this point?

File: e2e/tests/visual/planning.visual.spec.js (hunk 2 above)
Comment (ozyx) [reply e2e/tests/visual/planning.visual.spec.js:42]: Yeah! This is the same thing we do with the `tree` visual tests. Here we are only snapping the object view, so only the parts relevant to the Plan/Gantt Chart views

File: e2e/helper/faultUtils.js (hunk 3 above)
Comment (ozyx) [reply e2e/helper/faultUtils.js:61]: Probably. I think this example has been copied around and was originally used somewhere where we reused the `mainTree` locator.

**Pull request merged**